regression-tests/toh-contract-translation/Answers

```diff
@@ -0,0 +1,212 @@
+
+get-1.c
+Warning: no definition of function "non_det_int" available
+Warning: no definition of function "non_det_int" available
+Warning: no definition of function "non_det_int_ptr" available
+
+Inferred ACSL annotations
+================================================================================
+/* contracts for get */
+/*@
+  requires p == n && n_init >= 1;
+  ensures ((\result == 0 && 0 >= \old(*n)) || \result >= 0) && r1 == \old(r1) && n_init == \old(n_init) && \old(n) == n;
+*/
+================================================================================
+
+SAFE
+
+get-2.c
+Warning: no definition of function "non_det_int" available
+Warning: no definition of function "non_det_int" available
+Warning: no definition of function "non_det_int_ptr" available
+java.lang.Exception: Predicate generation failed
+(error "Predicate generation failed")
```

Comment:
I think this shouldn't be in `Answers` and in the list of tests until it reliably works.

Reply:
Right, I'll remove it then. Should I create an issue report to make sure this is not forgotten?

Reply:
This test case has been disabled.